packages/jest-preset/package.json

```diff
@@ -16,6 +16,6 @@
     "identity-obj-proxy": "^3.0.0"
   },
   "peerDependencies": {
-    "jest": "21"
+    "jest": "^21.0.0 || ^22.0.0"
```

Comment:
I'd prefer writing it in a way that the next major version doesn't require to add another `x.x.x || x.x.x || x.x.x`, but can just be extended, like so: `>= 21.0.0 < 23.0.0` or `21.x.x - 22.x.x`.